server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -632,6 +639,52 @@ private ClusterState applyCreateIndexRequestWithV2Template(
         );
     }
 
+    private ClusterState applyCreateIndexRequestForSystemIndex(
```

Comment:
At first, I used applyCreateIndexRequestWithV1Templates and a list of empty templates, but that seemed very unsatisfying.